Pod/API/PublicRemoteMethods/consoleCommand.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  consoleCommand.swift
+//  ElJayKit
+//
+//  Created by Isaac Greenspan on 12/14/15.
+//  Copyright © 2015 Isaac Greenspan. All rights reserved.
+//
+
+import Foundation
+import AlamofireXMLRPC
+
+extension API {
+    public enum ConsoleCommandOutputLineType: String {
+        case Normal = ""
+        case Info = "info"
+        case Error = "error"
+        case Success = "success"
+    }
+    
+    public struct ConsoleCommandOutputLine {
+        let type: ConsoleCommandOutputLineType
+        let text: String
+        
+        var display: String {
+            return NSString(format: "[%-7s] %@", (self.type.rawValue as NSString).UTF8String, self.text) as String
+        }
+    }
+    
+    public struct ConsoleCommandOutput {
+        let isSuccess: Bool
+        let output: [ConsoleCommandOutputLine]
+        
+        public var display: String {
+            return self.output.map({ $0.display }).joinWithSeparator("\n")
+        }
+    }
+    
+    public typealias ConsoleCommandCallback = Of<[ConsoleCommandOutput]>.callback
+    
+    public func consoleCommand(command command: String, callback: ConsoleCommandCallback) {
+        consoleCommand(commands: [command], callback: callback)
+    }
+    
+    public func consoleCommand(command command: [String], callback: ConsoleCommandCallback) {
+        consoleCommand(commands: [command], callback: callback)
+    }
+    
+    public func consoleCommand(commands commands: [String], callback: ConsoleCommandCallback) {
+        consoleCommand(
+            // Because Swift is stupid and can't turn [String] into [Any]...
```

Comment:
I should change the private method's parameter type from `[Any]` to `[AnyObject]` and then I'll be able to pass in the `[String]` and `[[String]]` types without any conversion?